tango/common/det_hash.py

```diff
@@ -82,9 +94,12 @@ def det_hash_object(self) -> Any:
             return None  # When you return `None` from here, it falls back to just hashing the object itself.
 
 
+_PICKLE_PROTOCOL = 4
```

Comment:
Can probably select protocol 5, since we require the minimum Python version to be 3.8.